src/components/missionComponents/CurrentMission.js

```diff
@@ -233,7 +248,12 @@ const CurrentMissionComponent = ({ user }) => {
   return (
     <Paper elevation={1} className={cx.topContainer}>
       {status === 'completed' ? (
-        <MissionComplete mission={currentMission} user={user} />
+        <MissionComplete
+          mission={currentMission}
+          user={user}
+          restartMission={restartMission}
+          showRestart
```

Comment:
we only show ability to restart on recently completed mission, not on past missions